meshcentral-config-schema.json

```diff
@@ -96,7 +96,7 @@
         "sqlite3": {
           "type": "boolean",
```

Comment:
dont forget to add the `string` type also to the schema file
```
"type": [
  "boolean",
  "string"
],
```

Reply:
Done.

Reply:
not quite fixed as you have set it to string,
my example above, you can set both a string or boolean
this is the proper way to do it

Reply:
Oops